### DISCUSSION TITLE
Problem z Skanerem Mail-owym

### DISCUSSION THREAD

**Discussion by mateusz-iwanski** [YetiForceCompany/YetiForceCRM#16204] [How to]
Witam, mam prawidłowo skonfigurowany SMTP jak i IMAP dla poczty, wszystko działa prawidłowo ale w momencie kiedy chcę ustawić skaner mailowy wyskakuje komunikat że nie może się połączyć z serwerem poczty. Usuwam konto, następnie ponownie loguje się prawidłowo wg instrukcji, komunikatu nie mam, przypisuje Akcje, chcę skonfigurować foldery i niestety się nie da, czekam nawet ponad godzinę na wyświetlenie się listy folderów ale bez skutecznie, po odświeżeniu strony znów pojawia się komunikat o nieprawidłowym logowaniu do serwera poczty.

**Comment by rskrzypczak** [comment]
You will probably find the answer here: https://github.com/YetiForceCompany/YetiForceCRM/issues/12589
